HW_1/task02.py

```diff
@@ -0,0 +1,11 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) 
+
+number = int(input('Введите число: '))
+result = 0
+while number > 0:
+    result = result + (number % 10)
```

Comment:
Тут скобки не нужны и можно записать покороче.
```suggestion
    result += number % 10
```